pkgs/applications/editors/neovim/default.nix

```diff
@@ -60,7 +61,7 @@ in
       neovimLuaEnv
       tree-sitter
       unibilium
-    ] ++ optional stdenv.isDarwin libiconv
+    ] ++ optional stdenv.isDarwin [ libiconv CoreServices ]
```

Comment:
Should be `optionals`

Reply:
Sorry about that, I amended the commit, hopefully that's what I was supposed to do.

I'm reading the documentation of `optional` and I don't understand why it worked locally without the change. Anyway, I tried again with `optionals` and it still builds

Reply:
Amending is correct